.gitignore

```diff
@@ -6,6 +6,7 @@
 # Build related
 tools/Cake/
 tools/NUnit.Runners/
```

Comment:
@devlead quick question before merging... Is this still required?  Is it not just NUnit.Console now?